pkgs/development/libraries/glib/default.nix

```diff
@@ -71,7 +71,9 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pkgconfig perl python gettext ];
 
-  propagatedBuildInputs = [ zlib libffi gettext libiconv ];
+  propagatedNativeBuildInputs = [ gettext ];
```

Comment:
I think this commit needs to go to staging.

Reply:
Also `gettext` can be removed from `nativeBuildInputs` now.

Reply:
Because you put it in `propagatedNativeBuildInputs` it will apply to `glib` as well.

Reply:
Ah, of course. Misread the first comment